napari/utils/misc.py

```diff
@@ -165,3 +165,16 @@ def __str__(self):
 
 
 callsignature = CallSignature.from_callable
+
+
+def all_subclasses(cls):
```

Comment:
Is `cls` reserved for class methods in python?

Reply:
`cls` is not reserved, it's just a convention (like `self`).  and as far as usage I have seen, it's a commonly used variable when referring to "some" class.

Reply:
Got it, thanks for the explanation!